tests/clothesOrder.spec.ts

```diff
@@ -0,0 +1,37 @@
+import { test, expect } from '@playwright/test'
+import { PageManager } from '../page-objects/pageManager'
+
+test.beforeEach(async ({ page }) => {
+    await page.goto('https://demo.prestashop.com/#/en/front')
+})
+
+test('Clothes Page', async ({ page }) => {
+    const pm = new PageManager(page)
+    pm.onClothesPage().getClothesPage()
+    //Asserting the Clothes page and clicking on the clothes page element
+    const clothesPage = await pm.onClothesPage().getClothesPage()
+    await clothesPage.click()
+    await clothesPage.textContent()
+    await expect(clothesPage).toContainText('Clothes')
+    // Scrolling to the element 
+    await pm.onClothesPage().getElementInput()
+    // Input of the number in field and assertion of the input data
+    const quantity = await pm.onClothesPage().getInputData()
+    await expect('1200').toEqual('1200')
+    // Adding items to cart
+    const addToCart = await pm.onClothesPage().getAddToCart()
+    // Proceeding to Checkout
+    await pm.onClothesPage().getProceedToCheckout()
+    // Clicking the arrowUp
+    await pm.onClothesPage().getArrowClick()
```

Comment:
This is just a suggestion 
- you can explain in comment what will be done (e.g. Increase product quantity by clicking on arrowUp)